app/views/recruitment_cycles/_courses_accredited_body.html.erb

```diff
@@ -1,6 +1,10 @@
 <% if @provider.accredited_body? && current_user["admin"]%>
   <div class="govuk-!-margin-bottom-8">
-    <h2 class="govuk-heading-m"><%= link_to "Courses as an accredited body","#", class: "govuk-link", data: { qa: 'courses_as_accredited_body_link' } %></h2>
+    <h2 class="govuk-heading-m"><%= link_to "Courses as an accredited body",
+                                    training_providers_provider_recruitment_cycle_path(@provider.provider_code,
+                                     year),
+                                     class: "govuk-link",
+                                     data: { qa: 'courses_as_accredited_body_link' } %></h2>
```

Comment:
At the risk of coming off as a white-space nazi, can we tidy this up to make it easier to parse?

```html.erb
    <h2 class="govuk-heading-m"><%= link_to "Courses as an accredited body",
                                            training_providers_provider_recruitment_cycle_path(
                                              @provider.provider_code,
                                              year
                                            ),
                                            class: "govuk-link",
                                            data: { qa: 'courses_as_accredited_body_link' } %></h2>
```

Reply:
Have done. Thanks